### ISSUE TITLE
Bug: My Home Black Friday card loses padding on <600px screens

### ISSUE THREAD

**Issue by inaikem** [Automattic/wp-calypso#96936]
### Quick summary

https://wordpress.com/home/DOMAIN_COM
The issue occurs as you go from 601 to 600, then to 599 px wide screens. 600 looks great but there’s some style loss as you go lower.

The `.task__black-friday-2024` container only has internal padding applied at from 600px wide screens and above:

```
@media(min-width: 600px) {
    .task .task__illustration,.task .task__text {
        padding:24px 16px
    }
}
```

### Steps to reproduce

The issue occurs as you go from 601 to 600, then to 599 px wide screens. 600 looks great but there’s some style loss as you go lower.

The .task__black-friday-2024 container only has internal padding applied at from 600px wide screens and above:

```
@media(min-width: 600px) {
    .task .task__illustration,.task .task__text {
        padding:24px 16px
    }
}
```

### What you expected to happen

Padding should be retained in some manner.

### What actually happened

See video:

https://github.com/user-attachments/assets/d1675298-048d-4208-bfaf-c96045fec1ce

### Impact

All

### Available workarounds?

No but the platform is still usable

**Comment by dzver** [comment]
I can only reproduce on users with the checklist complete. However, when the checklist is complete, it looks worse than this screenshot.

Regular user:
![Image](https://github.com/user-attachments/assets/881f1de2-6457-4308-baed-f88b45a1c831)

User with the checklist complete:
![Image](https://github.com/user-attachments/assets/b3a9a32f-4f07-4416-bf34-ffe81b88b6a9)

**Comment by dzver** [comment]
I confirmed that this bug can only happen if a person is resizing a browser to very small sizes. Looks fine on mobile.

**Comment by southp** [comment]
I can only reproduce the case that the right padding disappears, not the left one. I've tried that on the simulated devices on Chrome, my iPhone, and the responsive dev mode of Safari, but no luck. So my best guess is:

* At 600px, the layout container intentionally removes the left & right padding
![Image](https://github.com/user-attachments/assets/45d64534-5f01-40a7-af51-4de409948c9e)
* The dot-pager container expands to the full screen width at this point.
* The Black Friday banner by default occupies the whole container space.

That three of us can't exactly reproduce each other's issue likely shows that there is a race condition, resulting different layout according to the order of loading stylesheets. If we look at them individually, they all seem to do what they intend to do. Thus I'd need to experiment with all of them to see what's the right solution here. Tuning them individually without taking how the other styling rules get formed would properly make things worse.

Unfortunately somehow I have trouble identifying where the layout CSS is from. I have raised it here: p1733125557451119-slack-C02DQP0FP .

**Comment by Robertght** [comment]
@southp I can't replicate this anymore.

Video:

https://github.com/user-attachments/assets/08f1925a-dbd3-4ecf-946c-dd3a32b6a587

Is it safe to consider this fixed?

**Comment by wojtekn** [comment]
I couldn't reproduce it, too. The only way to reproduce it was manually disabling the margin set for the `.task` element:

![Image](https://github.com/user-attachments/assets/47b63427-36e9-4bec-ad56-e5702a94a3e7)

However, do we have a broader issue here? As far as I see, those are not aligned correctly in mobile view:
- "Our  Biggest Sale of the Year" task heading and content
- "Site setup" heading and items
- "Daily writing prompt" content
- "Get help" content
- "Quick links" heading and content

Also, shouldn't all those headings be the same size?

![Image](https://github.com/user-attachments/assets/3f2b58d7-a9bf-4dba-bf32-936d980e4d7c)

![Image](https://github.com/user-attachments/assets/e7fcbfba-ce38-4e07-a62d-bcc48c706d0d)

![Image](https://github.com/user-attachments/assets/b65158cd-0c6d-48c0-8977-ac0d98a5d3f3)

**Comment by annezazu** [comment]
This remains a problem for me when I look at my nomad.blog site. Can we fix this? It's not a good look so many days on, especially for a sale.

**Comment by dzver** [comment]
@jeyip could you have a look here, bring fresh eyes to this issue?

**Comment by m1r0** [comment]
I agree with @wojtekn, this could be a broader issue. I couldn't reproduce it with the Black Friday card but I'm getting it with the Quick Links one:

![Image](https://github.com/user-attachments/assets/5b850fc5-7751-415d-abcd-bcf01e3b6628)

**Comment by jeyip** [comment]
@dzver I also _cannot_ reproduce the padding issue. Here's what I tried:

- Personal Iphone 12 mini
- Chrome emulation with iphone 12 pro and iphone 14 pro
- Firefox emulation with iphone 12 pro and iphone 13 pro

I'm currently spinning up browserstack now to see if there's any luck.

**Comment by jeyip** [comment]
I also tested on a site that has recently undergone an atomic transfer. The padding continues to behave as expected. 🤔

**Comment by Robertght** [comment]
I was able to replicate the quick links widget flagged by @m1r0 

I also played a lot with the home widgets but I couldn't replicate it either. I even tried to remove the stats manually from the inspector thinking that would perhaps influence it, but it doesn't.

**Comment by supernovia** [comment]
@annezazu , @inaikem, and @m1r0  -- can you share what devices you're using when you see this issue?

I'm viewing on my iPhone with 390 x 657 and all of the padding looks normal whether I view with portrait or landscape, but I did see some odd padding issues when previewing with Inspect Element in Chrome and Safari.

**Comment by m1r0** [comment]
@supernovia Regarding the Quick Links padding issue I mentioned, it looks like it's happening only between 599px and 480px.

**Comment by supernovia** [comment]
@m1r0 Got it! I wasn't sure whether there are any phones/tablets that size or whether it only happens when testing with resizing browser windows.

**Comment by m1r0** [comment]
I've noticed a very similar with another task card:

![Image](https://github.com/user-attachments/assets/9923eb92-b1b4-4223-a363-a36c0edf8928)

Working on a fix in https://github.com/Automattic/wp-calypso/pull/97070.

**Comment by supernovia** [comment]
Awesome! I'll set the status to In Progress. If you need PA triage again though feel free to set back to Needs Triage.

**Comment by dzver** [comment]
Given that the Black Friday promotion ended last week. I'm not sure if the fix that's in progress will be relevant once the PR is ready.

**Comment by fredrikekelund** [comment]
Assigning this issue to you, @m1r0, since you wrote https://github.com/Automattic/wp-calypso/pull/97070

**Comment by fredrikekelund** [comment]
Did https://github.com/Automattic/wp-calypso/pull/97070 resolve this issue, too, @m1r0?

**Comment by m1r0** [comment]
I'm assuming so. I'm not able to reproduce the exact issue but it resolves a very [similar one](https://github.com/Automattic/wp-calypso/issues/96936#issuecomment-2521028766). Do you think we should mark this as resolved?

**Comment by annezazu** [comment]
I'll close this out for now. I don't have a card on my site so I can't confirm or deny if it's still happening. I will reopen if it does.
